purchase_amendment/data/purchase_workflow.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data noupdate="0">
+
+    <record id="trans_picking_except_picking_amend" model="workflow.transition">
+      <field name="act_from" ref="purchase.act_except_picking"/>
+      <field name="act_to" ref="purchase.act_picking"/>
+      <field name="signal">picking_amend</field>
```

Comment:
No one is calling this signal. IMHO this transition has no reason to exist. If the PO goes to "Except Picking" is because someone cancelled the picking. In that case, why would you need to amend the PO?